core/src/main/java/com/squarespace/template/plugins/platform/ContentFormatters.java

```diff
@@ -376,24 +383,6 @@ public void apply(Context ctx, Arguments args, Variables variables) throws CodeE
       buf.append("/>");
       var.set(buf);
     }
-
-    private String getAltText(Context ctx) {
```

Comment:
I removed this method because it seemed like it only existed for the case of image blocks, which are now handled on the item level. If this assumption is wrong, please let me know.